demos/falling/falling.go

```diff
@@ -91,21 +91,21 @@ func (control *ControlSystem) Update(entity *ecs.Entity, dt float32) {
 }
 
 type RockSpawnSystem struct {
-	*ecs.System
+	*ecs.BasicSystemWrapper
```

Comment:
I could rename `BasicSystemWrapper` to `BasicSystem`, and rename `BasicSystem` to `BasicSystemFunctions`? (since the latter will probably never be used, only implemented)

@paked ?